src/page.rs

```diff
@@ -86,11 +86,13 @@ impl Page {
         urls
     }
 
-    fn abs_path(&self, href: &str) -> Result<Url, ParseError> {
```

Comment:
You omited to remove `ParseError` import:

```
warning: unused import: `ParseError`
 --> src/page.rs:3:11
  |
3 | use url::{ParseError, Url};
  |           ^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: 1 warning emitted
```

I corrected this on 94c6486.

Reply:
Good catch.